tests/meta-mender-beaglebone-yocto-ci/conf/layer.conf

```diff
@@ -9,6 +9,6 @@ BBFILE_COLLECTIONS += "meta-mender-beaglebone-yocto-ci"
 BBFILE_PATTERN_meta-mender-beaglebone-yocto-ci = "^${LAYERDIR}/"
 BBFILE_PRIORITY_meta-mender-beaglebone-yocto-ci = "99"
 
-LAYERSERIES_COMPAT_meta-mender-beaglebone-yocto-ci = "dunfell"
+LAYERSERIES_COMPAT_meta-mender-beaglebone-yocto-ci = "dunfell kirkstone"
```

Comment:
We should remove `dunfell` from here. (we are not really using this file, but we should keep these consistent).

Reply:
Similarly for `meta-mender-raspberrypi3-ci`